src/code_80005FD0.c

```diff
@@ -91,17 +91,17 @@ u16 D_80163240[12];
 u16 D_80163258[12];
 u16 D_80163270[12];
 s32 D_80163288[10];
-UnkCommonTextureStruct0 *D_801632B0;
-u16 D_801632B8[12];
-u16 D_801632D0[12];
-u16 D_801632E8[12];
+CPUBehaviour *gActualCPUBehaviour;
```

Comment:
`gCurrentCPUBehaviour`?

Reply:
![image](https://github.com/n64decomp/mk64/assets/59367621/1a2319ce-91e2-4697-ab0f-1b6e1139a10f)
![image](https://github.com/n64decomp/mk64/assets/59367621/661e9e62-f216-498e-9d92-50f420b5fef8)
it's more a local variable and it's not per CPU it's just a place where you save the CPUBehaviour but it's not really global instead of gCurrentCPUBehaviourId

Reply:
`sCurrentCPUBehaviour`?